### DISCUSSION TITLE
Trying to Start PHP Server

### DISCUSSION THREAD

**Discussion by twalker-test** [yandeu/five-server#78] [Help / Questions] [answered]
I opened a termal in vscade and entered: 
php -S localhost:8000 -t www/ www/router.php

The terminal returned;
php : The term 'php' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the 
spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ php -S localhost:8000 -t www/ www/router.php
+ ~~~
    + CategoryInfo          : ObjectNotFound: (php:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException****

What is the proper command to be run in vscode to start the PHP server?

**Comment by yandeu** [comment, marked as answer]
The php cmdlet has nothing to do with VSCode.

Have a look at https://youtu.be/mVBe75aGBHQ

**Reply by twalker-test** [reply]
My environment variables for the path were not set up correctly. I also had to adjust the PHP server command to work on windows.
